Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1292,3 +1298,11 @@ void CGameListCtrl::UnselectAll()
 		SetItemState(i, 0, wxLIST_STATE_SELECTED);
 	}
 }
+bool CGameListCtrl::WiiCompressWarning(const GameListItem* iso)
+{
+	return iso->GetPlatform() == DiscIO::IVolume::WII_DISC &&
```

Comment:
Isn't this going to make the function return false for all GC discs, preventing users from compressing them?